docs/resources/openstack_vm.md

```diff
@@ -0,0 +1,753 @@
+
+For more details, see both the [CERN OpenStack][cern_openstack]{target=_blank .cern_login} website and its [documentation][cern_openstack_doc]{target=_blank} pages.
+
+### What to expect
+
+After following this guide you will have your own virtual machine running on CERN OpenStack.
+This machine will be similar to your office desktop or the `lxplus` machines,
+with access to `afs`, `eos` and `htcondor` yet come with some advatages and disadvantages:
+
+**Advantages:**
+
+- Full root access to your machine.
+- Independence from Building 6/9/10 powercuts (in comparison to your office desktop machine).
+- Customizable hardware (RAM, Disk, CPU).
+- Persistent storage (your machine will stay alive until you delete it).
+- No interference from other users (your machine is yours alone).
+- Ability to install custom software.
+- Ability to run long-running processes without being logged in.
+- Ability to restart the machine whenever needed.
+- Large shareable local storage (if you mount a volume).
+
+**Disadvantages:**
+
+- No automatic updates (you have to maintain your machine yourself).
+- No external access, only via CERN network or lxplus-tunneling.
+- Not a high-performace machine (compared to dev/optics servers and HPC clusters), especially when not part of a [project](#projects).
+
+### Projects
+
+CERN OpenStack allows users to create virtual machines either as individuals or as part of a [project][cern_openstack_doc_projects]{target=_blank}.
+Within a project more flavours and larger resource quotas are available, making it possible to create more powerful virtual machines.
+
+!!! warning "Project Request"
+    You need to request a project through the CERN IT department directly from the [OpenStack portal][cern_openstack]{target=_blank}
+    **after confirming that you are allowed to do so with the person responsible for computing resources in your group** (currently _G. Iadarola_ for ABP).
+
+!!! tip "OMC Project"
+    OMC members can request to join the existing **OMC OpenStack project** (`omc.cern.ch` and `omc1.cern.ch`) instead of creating a new one.
+    This way the resource usage can be shared among all OMC members.
+    Members can be added by the administrators of the project upon request.
+    To become an administrator of the OMC project yourself, please request to be added to the [omc-openstack-admins group][omc_openstack_admins]{target=_blank} by _E. Maclean_.
+
+## Virtual Machine Setup
+
+This setup guide is very similar to the [Creating a CentOS CS8 Virtual Machine][openstackCS8]{target=_blank},
+but held a bit more generic to allow for different operating systems.
+
+If you want to use a non-RHEL-based operating system (e.g. Ubuntu),
+you need to adapt the installation steps, e.g. replacing `dnf`/`yum` with `apt` etc.
+
+### Create SSH Key pair
+
+Create an ssh-public-private-key pair for authentication.
+
+```bash
+ssh-keygen -t rsa -f your-key-name
```

Comment:
It seems to me `rsa` is no longer the recommended encryption for these, and `ed25519` or others are preferred? Iirc from the GitHub docs when I created some pairs.

Does CERN Openstack request RSA encryption?